server/chroma/src/routes/appdata.rs

```diff
@@ -16,4 +24,20 @@ pub struct AppData {
     pub storage: Storage,
     pub config: Config,
     pub koala: KoalaApi,
+    pub ratelimits: Ratelimits,
+}
+
+#[derive(Debug, Clone)]
+pub struct Ratelimits {
```

Comment:
Is this the most logical place to put the Ratelimits struct?

Reply:
Sure, its part of the application's state and can be expanded for more things in the future.

Reply:
Good enough for me